assets/xml/objects/object_oyu.xml

```diff
@@ -1,9 +1,10 @@
 ﻿<Root>
+    <!-- Hot Spring Water found in the Goron Graveyard found in Snowhead Village -->
     <File Name="object_oyu" Segment="6">
-        <DList Name="object_oyu_DL_000080" Offset="0x80" />
-        <DList Name="object_oyu_DL_000158" Offset="0x158" />
-        <Texture Name="object_oyu_Tex_000160" OutName="tex_000160" Format="rgba16" Width="32" Height="32" Offset="0x160" />
-        <TextureAnimation Name="object_oyu_Matanimheader_000968" Offset="0x968" />
-        <Collision Name="object_oyu_Colheader_000988" Offset="0x988" />
+        <DList Name="gHotSpringWaterDL" Offset="0x80" />
+        <DList Name="gHotSpringWaterEmptyDL" Offset="0x158" />
```

Comment:
```suggestion
        <DList Name="gGoronGraveyardHotSpringWaterDL" Offset="0x80" />
        <DList Name="gGoronGraveyardHotSpringWaterEmptyDL" Offset="0x158" />
```
I think adding the `GoronGraveyard` prefix to this was a good suggestion, could you add to the names of this file?